File: qlty-cli/tests/lang/go/basic.in/functions/methods_without_parameters.go
type Rectangle struct {
	width  float64
	height float64
}

func NewRectangle() *Rectangle {
	return &Rectangle{
		width:  1,
		height: 2,
	}
}

func (r *Rectangle) Area() float64 {
	return r.width * r.height
}

func (r *Rectangle) Foo() {
}

func (r *Rectangle) Bar() {
}

File: qlty-cli/tests/lang/typescript/basic.in/functions/methods_without_parameters.ts
class Rectangle {
  width: number;
  height: number;

  constructor(width: number, height: number) {
    this.width = width;
    this.height = height;
  }

  area(): number {
    return this.width * this.height;
  }

  foo(): void {}

  bar(): void {}
}
